Solutions/Salesforce Service Cloud/Analytic Rules/Salesforce-SigninsMultipleCountries.yaml

```diff
@@ -0,0 +1,43 @@
+id: 3094e036-e5ae-4d6e-8626-b0f86ebc71f2
+name: Brute force attack against user credentials
+description: |
+  'This query searches for successful user logins from different countries within 3 hours.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: SalesforceServiceCloud
+    dataTypes:
+      - SalesforceServiceCloud
+queryFrequency: 20m
+queryPeriod: 20m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1078
+query: |
+  let timeframe = ago(3h);
```

Comment:
The queryFrequency and queryPeriod are set to 20m, but you have a timeframe set.  You should not do this unless there is a required lookback for comparison, there is not one here.

Reply:
Fixed